File: caitlin-schaeffer/day4/centuryFromYear.js
/* Introduction
The first century spans from the year 1 up to and including the year 100, the second century - 
from the year 101 up to and including the year 200, etc.

Params: Given a year, return the century it is in. 
Return: integer (what century a year falls in)
Concerns: for the life of me I have never figured out centuries
Solution: century can be calculated by dividing the year by 100 and rounding up. Use the Math.ceil function.

Original provided code:
function century(year) {
  // Finish this :)
  return;
}

*/


function century(year) {
    let century = Math.ceil(year/100)
     return century
   }
 

File: caitlin-schaeffer/day-31/century-from-year.py
# Instructions: Given a year, return the century it is in.
# Examples: 1705 --> 18
# Params: given integer
# Return: return integer
# Concerns: none
# Solution: simply divide the year by 100 and use math ceil to round the number up to it's nearest integer

# Given Code: 
# def century(year):
#     # Finish this :)
#     return



import math

def century(year):
    return math.ceil(year/100)